### ISSUE TITLE
cannot create audio tour

### ISSUE THREAD

**Issue by lseiter** [RunestoneInteractive/RunestoneComponents#1043]
Audio files do not play when creating an audio tour.  The component code in
https://github.com/RunestoneInteractive/RunestoneComponents/blob/master/runestone/activecode/js/audiotour.js 
expects audio files to be stored in media.interactivepython.org rather than the directory static/audio (commented out code).
I do not have access to store files on media.interactivepython.org

My sample code:

.. activecode:: testaudio
   :language: java
   :tour_1: "Overall Tour"; 1: test_1a; 2: test_1b; 

   Testing audio tour
   ~~~~
   public class TestAudio
   {
      public static void main(String[] args)
      {
         System.out.println("Hello");
         System.out.println("Welcome to Java");
      }
   }
           
   ====

**Comment by bnmnetp** [comment]
Fixed in #1035

**Comment by bnmnetp** [comment]
Fixed in release 5.0.13
